src/store/login/saga.ts

```diff
@@ -167,6 +168,9 @@ export function* openFirstConversationAfterChannelsLoaded() {
   const isMessengerFullScreen = yield select((state) => getDeepProperty(state, 'layout.value.isMessengerFullScreen'));
   if (payload.loaded && isMessengerFullScreen) {
     yield call(openFirstConversation);
+
+    yield delay(2000);
+    yield call(checkNewRewardsLoaded);
```

Comment:
Hmm.. is there a reason to delay this? Don't we want to just render this right away if they have new rewards when they log in? As in, is there even a reason to wait for the conversations to load?

Reply:
no reason, i just thought it would look a little nicer on the UI. i.e the first conversation loads up (takes a second or two since it's a sendbird call), and then popup the rewards tooltip. 

https://github.com/zer0-os/zOS/assets/33264364/27c1d80f-9b02-4b17-9de7-8295e9531b3e